experimenter/tests/integration/nimbus/android/fenix-build.env

```diff
@@ -0,0 +1 @@
+FIREFOX_FENIX_BUILD_ID=Nm06KKu8SBypTeRBUxZYvQ
```

Comment:
nit: this should be called a task id because thats what it is. Build ID is a thing fore Firefox and looks like YYYYMMDD.TTTTTT or something